brainglobe_atlasapi/atlas_generation/atlas_scripts/allen_mouse_bluebrain_barrels.py

```diff
@@ -0,0 +1,311 @@
+__version__ = "2"
```

Comment:
This should be 0. This is the atlas/minor version, that will get combined with the current version of the API to create the final version of the packaged atlas. If this is set to 0, then the final packaged atlas will be 1.0. It can be incremented each time the script is updated. If the atlas API is updated, it will jump to 2.0.  
```suggestion
__version__ = "0"
```

Reply:
This was fixed